airflow_provider_fivetran/hooks/fivetran.py

```diff
@@ -1,23 +1,24 @@
+import json
+import logging
 from time import sleep
-from urllib.parse import urlparse
 
 import requests
 from requests import PreparedRequest, exceptions as requests_exceptions
 from requests.auth import AuthBase
 import pendulum
-import logging
-import json
-import time
-
 
-from airflow import __version__
+# from airflow import __version__
 from airflow.exceptions import AirflowException
-from airflow.hooks.base_hook import BaseHook
+from airflow.hooks.base import BaseHook
```

Comment:
This suppresses a deprecation warning from Airflow.

Reply:
@pgzmnk Question for you, is there any danger to replacing this here? Looking at the code, it looks like it does a logging mixin and that is about it.